examples/demo_mult_image_sockets2.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
```

Comment:
Similar to my comment on the other example with sockets this functionality seems too advanced for our examples right now. It was cool to watch the animation, and in the future we can think about more animation / multiprocess examples, but I think you should maintain this separately for now